File: user/src/main/java/com/kjj/user/dto/user/MyPageDto.java
package com.kjj.user.dto.user;

import com.kjj.user.entity.UserMyPage;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@AllArgsConstructor
@NoArgsConstructor
public class MyPageDto {
    private Long userId;
    private int point;

    public static MyPageDto from(UserMyPage userMyPage) {
        return new MyPageDto(
                userMyPage.getId(),
                userMyPage.getPoint()
        );
    }
}

File: user/src/main/kotlin/com/kjj/user/dto/user/MyPageDto.kt
package com.kjj.user.dto.user

import com.kjj.user.entity.UserMypage

data class MyPageDto(
    val userId: Long?,
    val point: Int
) {
    companion object {
        fun from(userMyPage: UserMypage): MyPageDto {
            return MyPageDto(
                userMyPage.id,
                userMyPage.point
            )
        }
    }
}